### PR TITLE
Fix attachments outputs

### PR DESCRIPTION
**Pull request by mariux**
- Rename out put to policy_attachments and include all attachment objects
- Prepare 0.4.0 release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ The following attributes are exported by the module:
 
 - **`role`**: The `aws_iam_role` object.
 - **`policy`**: The `aws_iam_role_policy` object.
-- **`policy_attachment`**: The `aws_iam_role_policy_attachment` object.
+- **`policy_attachments`**: An Array of `aws_iam_role_policy_attachment` objects.
```
Comment (soerenmartius) [inline README.md:266]:
```suggestion
- **`policy_attachments`**: An array of `aws_iam_role_policy_attachment` objects.
```

**Review by soerenmartius — APPROVED** [review]
please consider the change. LGTM!

**Comment by mariux** [comment]
@soerenmartius thx.. fixed..

**Review by soerenmartius — APPROVED** [review]
lgtm!

**Pull request merged**